pom.xml

```diff
@@ -80,6 +80,11 @@
                 <scope>import</scope>
                 <type>pom</type>
             </dependency>
+            <dependency>
+                <groupId>org.jenkins-ci.plugins.workflow</groupId>
+                <artifactId>workflow-step-api</artifactId>
+                <version>999999-SNAPSHOT</version>
```

Comment:
requires https://github.com/jenkinsci/workflow-step-api-plugin/pull/204 testing in local with `mvn clean install`

Reply:
nice,. fixed now. ty